src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCardHook.js

```diff
@@ -26,16 +28,23 @@ export const useScenarioDashboardCard = () => {
   );
 
   const isDashboardSync = useMemo(() => {
-    if (!currentScenarioRunStartTime) return true;
+    const disableOutOfSyncWarningBanner = (workspace?.webApp?.options?.disableOutOfSyncWarningBanner ?? false) === true;
```

Comment:
In which case we could need this nullish coalescing operator ? if `disableOutOfSyncWarningBanner` isn't defined in the workspace, `undefined === true` will return false, no ? :thinking:

Reply:
Oops, you're right, I'll remove it :smile:  Thank you !